app/src/components/file-upload/FileUploadItem.tsx

```diff
@@ -208,196 +284,73 @@ const FileUploadItem: React.FC<IFileUploadItemProps> = (props) => {
-                <Typography variant="caption" component="div">
-                  {error ? (
-                    <>
-                      {error}
-                      {props.enableErrorDetails && errorDetails && errorDetails.length ? (
-                        <IconButton size="small" onClick={() => setShowErrorDetails(!showErrorDetails)}>
-                          {showErrorDetails ? (
-                            <SvgIcon>
-                              <path d={mdiChevronUp} />
-                            </SvgIcon>
-                          ) : (
-                            <SvgIcon>
-                              <path d={mdiChevronDown} />
-                            </SvgIcon>
-                          )}
-                        </IconButton>
-                      ) : null}
-                      <Collapse in={showErrorDetails}>
-                        <Typography variant="caption" component="div" color="error">
-                          <List dense>
-                            {errorDetails?.map((detail) => (
-                              <ListItem key={detail._id}>{detail.message}</ListItem>
-                            ))}
-                          </List>
-                        </Typography>
-                      </Collapse>
-                    </>
-                  ) : (
-                    status
-                  )}
-                </Typography>
-              </Box>
-              <Box display="flex" alignItems="center">
-                <MemoizedActionButton status={status} onCancel={() => setInitiateCancel(true)} />
-              </Box>
-            </Box>
-            <MemoizedProgressBar status={status} progress={progress} />
-          </Box>
-        </Box>
+    <ListItem
+      key={file.name}
+      secondaryAction={<MemoizedActionButton status={status} onCancel={() => setInitiateCancel(true)} />}
+      sx={{
+        flexWrap: 'wrap',
+        borderStyle: 'solid',
+        borderWidth: '1px',
+        borderRadius: '6px',
+        background: grey[100],
+        borderColor: grey[300],
+        '& + li': {
+          mt: 1
+        },
+        '& .MuiListItemSecondaryAction-root': {
+          top: '36px'
+        },
+        '&:last-child': {
+          borderBottomStyle: 'solid',
+          borderBottomWidth: '1px',
+          borderBottomColor: grey[300]
+        }
+      }}>
+      <ListItemIcon
+        sx={{
+          '&.fileIconColor': {
+            color: 'text.secondary'
+          },
+          '&.error': {
+            color: 'error.main'
+          }
+        }}>
+        <Icon path={mdiFileOutline} size={1.25} className={error ? 'errorColor' : 'fileIconColor'} />
+      </ListItemIcon>
+      <ListItemText
+        primary={file.name}
+        secondary={
+          <Subtext
+            file={file}
+            status={status}
+            progress={progress}
+            error={error}
+            errorDetails={[{ _id: '123123', message: 'awdawdawdawdaw' }]}
```

Comment:
Nit: leave these as empty strings so dummy values are not accidentally exposed

Reply:
Ah, this was probably me forgetting test data

Reply:
Fixed